routes/reviews.js

```diff
@@ -19,11 +19,12 @@ const payload = await getAverageRating(cmsAdapter, req.params.id);
 });
 
 reviewRouter.post('/reviews', async (req, res) => {
-  try {
-    await postReview(cmsAdapter, req.body);
-    res.status(200).send({ message: 'Success' });
-  } catch (error) {
-    res.status(400).send({ message: error.message });
+  const validated = await validateReview(req.body);
+  if (validated === false) {
+    res.status(400).send({ message: 'Invalid input' });
+  } else {
+  await postReview(cmsAdapter, req.body);
+  res.send({ message: 'Success' });
```

Comment:
The code style here is a little bit sloppy, with indentation not matching the block that the code is in. I would expect the code on these two lines to be indented one more level.